CourtFinder/endpoints/users/routes.py

```diff
@@ -80,3 +80,59 @@ def register():
             return render_template('users/profile.html', user=new_user)
 
     return render_template('users/register.html', form=form)
+<<<<<<< HEAD
+
+
+@users.route('/UpdateProfile/<id>', methods=['GET','POST'])
+def updateProfile(id):
+    if request.method == 'POST':
+        user = User.query.filter_by(id=id).first()
+        user.first_name = request.form.get('inputFirstName')
+        user.last_name = request.form.get('inputLastName')
+        user.favorite_court = request.form.get('inputFavoriteCourt')
+        user.height = request.form.get('inputHeight')
+        user.weight = request.form.get('inputWeight')
+        user.skill_level = request.form.get('inputSkillLevel')
+        db.session.commit()
+    if reguest.form.get('inputPassword') == None and request.form.get('inputConfirmPassword') == None:
+        flash('Password was not changed', 'danger')
+    elif request.form.get('inputPassword') == request.form.get('inputConfirmPassword'):
+        hashed_pass = sha256_crypt.encrypt(str(request.form.get('inputPassword')))
+        user.password= hashed_pass
+        db.session.commit()
+    else:
+        flash('Passwords dont match', 'danger')
+
+        if not check_username(request.form.get('inputUserName')):
```

Comment:
this looks indented too much, line 106 - 117 will only run if passwords do not match